utils/_context/_scenarios/__init__.py

```diff
@@ -453,13 +453,18 @@ def all_endtoend_scenarios(test_object):
             "DD_APPSEC_ENABLED": "false",
             "DD_TRACE_HTTP_CLIENT_ERROR_STATUSES": "200-201,202",
             "DD_SERVICE": "service_test",
+            "DD_TRACE_Kafka_ENABLED": "false",
         },
+        include_kafka=True,
         doc="",
         scenario_groups=[ScenarioGroup.ESSENTIALS],
     )
 
     tracing_config_nondefault_3 = EndToEndScenario(
-        "TRACING_CONFIG_NONDEFAULT_3", weblog_env={"DD_TRACE_HTTP_CLIENT_TAG_QUERY_STRING": "false"}, doc="",
+        "TRACING_CONFIG_NONDEFAULT_3",
+        weblog_env={"DD_TRACE_HTTP_CLIENT_TAG_QUERY_STRING": "false", "DD_TRACE_Kafka_ENABLED": "true"},
```

Comment:
nit: casing

```suggestion
        weblog_env={"DD_TRACE_HTTP_CLIENT_TAG_QUERY_STRING": "false", "DD_TRACE_KAFKA_ENABLED": "true"},
```

Reply:
Indeed so the other languages need to add the env var as their tracer expects it as they enable the test(some languages may even have different integrations names).

Reply:
@zacharycmontoya Well, the casing is required for the PHP extension to recognize the flag on Linux, but in uppercase.
If I read it correctly at least Java and Ruby are using full uppercase too. From a quick glance at docs I couldn't determine what exactly Python and node are doing.
So... we have to change this in any case.
I'd propose to universally move to fully uppercase.

Reply:
> I'd propose to universally move to fully uppercase. I find uppercase also more intuitive, in the midst of everything else being uppercase.

Agree, all our configuration env var are upper case. It looks weird to have one that mix both. By the why, is there a RFC for that ?

Reply:
So it was previously implemented like that, not sure I could find the RFC for that since is so old, but we have agreed that for these efforts of the [Config Consistency RFC](https://docs.google.com/document/d/1kI-gTAKghfcwI7YzKhqRv2ExUstcHqADIWA4-TZ387o/edit#heading=h.8v16cioi7qxp) it should be added that we always expected a all Upper cased env var.